src/Pelagos/Bundle/AppBundle/Resources/views/Account/EmailNotFound.html.twig

```diff
@@ -1,10 +1,9 @@
 {% extends "PelagosAppBundle:template:UI.html.twig" %}
 
 {% block body %}
-    <div class="messages error alert alert-warning">
+    <div class="messages error">
         <h2 class="element-invisible">Error message</h2>
-        <i class="fas fa-exclamation-circle"></i>
         The email address you have entered could not be found in the GRIIDC system. Please contact <a href=
-        "griidc@gomri.org" class="alert-link">griidc@gomri.org</a> for additional help.
+        "griidc@gomri.org">griidc@gomri.org</a> for additional help.
```

Comment:
I guess this is how it was in production too. Should we fixed this links, I think we should... Add email: before the e-mail addres.

Reply:
And off course it's "mailto:"...
https://www.iana.org/assignments/uri-schemes/uri-schemes.xhtml